src/whyDidYouRender.js

```diff
@@ -71,6 +71,15 @@ function trackHookChanges(hookName, { path: hookPath }, hookResult) {
     });
 
     if (notification.reason.hookDifferences) {
+      if (wdyrStore.options.showLineHook) {
+        try {
+          throw new Error().stack.split(' at ')
+            .find(line => line.match(displayName)).split('/').pop();
+        } catch (error) {
+          const lineHook = '(' + error;
+          notification.lineHook = lineHook;
+        }
```

Comment:
I don't think you actually need to throw the error to get the stack trace. 
```suggestion

          const matchedLine = new Error().stack.split(' at ')
            .find(line => line.match(displayName)).split('/').pop();

          const lineHook = '(' + error;
          notification.lineHook = lineHook;

```

Reply:
Nice, don't need to use try catch.